internal/controller/argocd/api/v1alpha1/application_types.go

```diff
@@ -66,7 +67,10 @@ const (
 )
 
 type ApplicationSourceHelm struct {
-	Parameters []HelmParameter `json:"parameters,omitempty"`
+	ReleaseName  string          `json:"releaseName,omitempty"`
+	ValueFiles   []string        `json:"valueFiles,omitempty"`
+	ValuesObject any             `json:"valuesObject,omitempty"`
```

Comment:
@hiddeco , `codegen` is complaining about the `any` type of this field. I'm not familiar with the constraints of codegen, what should be used here?